sympy/physics/control/lti.py

```diff
@@ -224,6 +291,22 @@ def var(self):
         """
         return self._var
 
+    # TODO: Remove num_inputs, num_outputs and shape from TF
```

Comment:
These would likely have to follow a deprecation: https://github.com/sympy/sympy/wiki/Deprecating-policy

Reply:
Since it has not been released yet, should I write a deprecation warning for it?
Even on the wiki page, it's mentioned:
> Note: Code that has not yet made it into a release does not need to be deprecated.

Reply:
If it hasn't been released, then no. I thoutht this was TransferFunction code.